### PR TITLE
Copy interval docs from grdcontour.py to the levels docs in contour.py

### PR DESCRIPTION
**Pull request by willschlitzer**
As pointed out by @meghanrjones in #1042, the parameter `levels` in `contour.py` is the same as `interval` in `grdcontour.py`. The `interval` documentation was much more complete than the `levels` documentation; this pull request copies the documentation from `interval` to `levels`.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/src/contour.py [hunk 1]
```diff
@@ -64,8 +64,16 @@ def contour(self, x=None, y=None, z=None, data=None, **kwargs):
         - Specify a fixed annotation interval *annot_int* or a
           single annotation level +\ *annot_int*.
     {B}
-    levels : str
-        Contour file or level(s)
+    levels : str or int
+        Specify the contour lines to generate.
+
+        - The filename of a `CPT`  file where the color boundaries will
```
Comment (seisman) [inline pygmt/src/contour.py:70]:
```suggestion
        - The filename of a CPT file where the color boundaries will
```

Comment (seisman) [reply pygmt/src/contour.py]: @willschlitzer Could you also make the same fix in `grdcontour.py`?

Comment (willschlitzer) [reply pygmt/src/contour.py]: Done in 50fbf4e

**Comment by meghanrjones** [comment]
Is the no approval for this and #1044 a GitHub bug or a force merge?

**Comment by seisman** [comment]
It's a simple fix so I just pushed the "force merge" button (after reviewing the changes 😃)

**Comment by meghanrjones** [comment]
> It's a simple fix so I just pushed the "force merge" button (after reviewing the changes 😃)

OK, sounds good. I just wanted to make sure we weren't have any GitHub problems after the issues yesterday.

**Pull request merged**